cypress/support/settings.ts

```diff
@@ -72,18 +78,30 @@ Cypress.Commands.add("openUserSettings", (tab?: string): Chainable<JQuery<HTMLEl
     });
     return cy.get(".mx_UserSettingsDialog").within(() => {
         if (tab) {
-            cy.switchTabUserSettings(tab);
+            cy.switchTab(tab);
+        }
+    });
+});
+
+Cypress.Commands.add("openRoomSettings", (tab?: string): Chainable<JQuery<HTMLElement>> => {
+    cy.get(".mx_RoomHeader_name").click();
+    cy.get(".mx_RoomTile_contextMenu").within(() => {
+        cy.get('[aria-label="Settings"]').click();
```

Comment:
Shouldn't we not better introduce some `data-test-id` attributes? If you find a `data-test-id` it would be clear what it is doing. While I won't expect tests to break if I change an aria label.

Reply:
I'm personally a fan of exercising the aria attributes as part of the tests, this way you make a11y related changes require confirmation by means of updating the test. In the end, the aria label is part of how an AT-user will navigate the app, which is more important than some hidden element attribute.